maintenance/README.md

```diff
@@ -238,7 +238,21 @@ Indicated by this error:
     this should be done)
 
 
-## 6) Build Stubs
+## 6) Build Stubs (new)
+
+  - Fix up the mypy typeshed. This is only required until my fixes to mypy stubgen
+    get officially released (at which point the requirements.txt file in this
+    directory will be updated and this note will be removed:
+    ```
+    git clone http://github.com/python/typeshed $(mayapy -c "import mypy,os;print(os.path.join(mypy.__path__[0], 'typeshed'))"
+    ```
+
+  - run `stubgen`:
```

Comment:
Think this should be either `genstubs` (to match below) or `mypy.stubgen`

Reply:
fixed